### PR TITLE
log exception information while decrypt in cookie selector

### PR DESCRIPTION
**Pull request by barakpx**

### CODE REVIEW

File: src/main/java/com/perimeterx/internals/CookieSelector.java [hunk 1]
```diff
@@ -116,11 +116,11 @@ private static String evaluateOriginalTokenCookie(AbstractPXCookie selectedCooki
         } else {
             try {
                 if (!selectedCookie.deserialize()) {
-                    logger.debug("Original token decryption failed, value: " + selectedCookie.getPxCookie());
+                    logger.debug("Original token decryption failed, value: {}", selectedCookie.getPxCookie());
                     error = "decryption_failed";
                 }
             } catch (PXCookieDecryptionException e) {
-                logger.debug("Original token decryption failed, value: " + selectedCookie.getPxCookie());
+                logger.debug("Original token decryption failed with exception, value: {}", selectedCookie.getPxCookie(), e);
```
Comment (pxjohnny) [inline src/main/java/com/perimeterx/internals/CookieSelector.java]: shouldnt this be `logger.error`?

File: src/main/java/com/perimeterx/internals/CookieSelector.java [hunk 2]
```diff
@@ -97,11 +97,11 @@ private static S2SCallReason evaluateCookie(AbstractPXCookie selectedCookie, Str
         } else {
             try {
                 if (!selectedCookie.deserialize()) {
-                    logger.debug("Cookie decryption failed, value: " + cookie);
+                    logger.debug("Cookie decryption failed, value: {}", cookie);
                     s2SCallReason = S2SCallReason.INVALID_DECRYPTION;
                 }
             } catch (Exception e) {
-                logger.debug("Cookie decryption failed, value: " + cookie);
+                logger.debug("Cookie decryption failed with exception, value: {}", cookie, e);
```
Comment (pxjohnny) [inline src/main/java/com/perimeterx/internals/CookieSelector.java]: shouldnt this be `logger.error`?

Comment (barakpx) [reply src/main/java/com/perimeterx/internals/CookieSelector.java]: prevent log filled with error by attacker with invalid cookie

File: src/main/java/com/perimeterx/internals/CookieSelector.java (hunk 1 above)
Comment (barakpx) [reply src/main/java/com/perimeterx/internals/CookieSelector.java]: prevent log filled with error by attacker with invalid cookie

**Comment by noamlevipx** [comment]
pls change the target branch to dev

**Pull request merged**